### ISSUE TITLE
Changing brightness does not put EC in blocked state (3.2 broken)

### ISSUE THREAD

**Issue by broyuken** [danobot/entity-controller#174]
<<< Do not delete the following template headers. All information must be filled in because we need it to analyse the problem. Your issue will be auto closed without this information.
Before opening a new issue, please read the documentation and search for keywords relating to your problem. Since this is an open source project expect to be involved in the issue resolution because we depend on your support. >>>

## Description

Is 3.2 still broken? I am just trying to implement it myself now, and if I manually change the brightness either via HA or via the wall it does not put it into a blocked state. The timer still shuts off after no motion.

## Configuration
Include configuration of affected entity.

## Steps to reproduce
Trigger EC, adjust brightness, light still turns off after timer. 

### Expected behavior
Light stays on after timer expires after being manually adjusted. 

### Actual Behaviour
Light turns off after I adjust brightness manually. 
## Logs
Provide logs at DEBUG level. 

## Version
If you are not using the latest version, please upgrade first.
Latest

**Comment by danobot** [comment]
Can you supply a sample entity that reproduces this problem?

**Comment by broyuken** [comment]
entity_controller:
  ec_office_light:
    sensor: binary_sensor.office_motion
    entity: light.office_light
    delay: 600

**Comment by danobot** [comment]
I think the problem is [L308](https://github.com/danobot/entity-controller/blob/f2fd1b99a4e4d59b7b6775d00f323416decf568a/custom_components/entity_controller/__init__.py#L308) which has a commented transition which is called on [L649](https://github.com/danobot/entity-controller/blob/f2fd1b99a4e4d59b7b6775d00f323416decf568a/custom_components/entity_controller/__init__.py#L649).

I uncommented this and added the fix onto the `develop` branch for trial and review. Please load the [`__init__.py` file from develop branch](https://github.com/danobot/entity-controller/blob/develop/custom_components/entity_controller/__init__.py) and retest.

**Comment by broyuken** [comment]
Thanks for the attempt, but it does not seem to be working. It just immediately went into a blocked state without me changing brightness or anything. 

```

sensor type
event
delay
10s
last triggered by
binary_sensor.office_motion
last triggered at
2020-08-31 10:53:47.112685
expires at
2020-08-31T10:53:57.112989
service data

entity_id: light.office_light

blocked at
2020-08-31T10:53:47.455038
blocked by
light.office_light
```

**Comment by danobot** [comment]
Can you provide the debug log?
Its running fine in my instance and on the demo instance.

**Comment by broyuken** [comment]
turning on debug now. What state does your light show when it is first turned on, but before it is manually adjusted? does it show blocked or active/timer?

**Comment by broyuken** [comment]
`2020-09-01 06:53:03 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for entity_controller which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant.
2020-09-01 06:53:12 INFO (MainThread) [custom_components.entity_controller] If you have ANY issues with EntityController (v6.1.0), please enable DEBUG logging under the logger component and kindly report the issue on Github. https://github.com/danobot/entity-controller/issues
2020-09-01 06:53:12 INFO (MainThread) [custom_components.entity_controller] Domain Configuration: OrderedDict([('ec_office_light', OrderedDict([('sensor', 'binary_sensor.office_motion'), ('entity', 'light.office_light'), ('delay', 600)]))])
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller] Setting up entity services
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Initialising EntityController entity with this configuration: 
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] OrderedDict([('sensor', 'binary_sensor.office_motion'),
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Controller name: ec_office_light
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Control Entities: ['light.office_light']
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Added Control Entities as state entities: ['light.office_light']
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Sensor Entities: ['binary_sensor.office_motion']
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Transition Behaviours: {'on_enter_active': 'on',
2020-09-01 06:53:12 INFO (MainThread) [custom_components.entity_controller.ec_office_light] Service data set up
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Ignoring state changes on the following attributes: []
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] --------------------------------------------------
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light]        C O N F I G U R A T I O N   D U M P        
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] --------------------------------------------------
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Entity Controller       ec_office_light
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Sensor Entities         ['binary_sensor.office_motion']
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Control Entities:       ['light.office_light']
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State Entities:         ['light.office_light']
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Activate Trigger E.:    []
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Deactivate Trigger E.:  []
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Ignored state attrs:    []
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Light params:           {}
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light]         -------        Time        -------        
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Start time:             None
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] End time:               None
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] DT Now:                 2020-09-01 06:53:12.041740-04:00
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] datetime Now:           2020-09-01 06:53:12.043007
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Next Sunrise:           2020-09-02 06:09:31-04:00
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Next Sunset:            2020-09-01 19:23:29-04:00
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light]         -------        Sun         -------        
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Sunrise:                2020-08-31 06:09:31-04:00
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Sunset:                 2020-08-31 19:23:29-04:00
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Sunset Diff (to now): 12:30:16.924217
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Sunrise Diff(to now): 12:30:16.922223
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Transition Behaviours: {'on_enter_idle': 'off', 'on_exit_idle': 'ignore', 'on_enter_active': 'on', 'on_exit_active': 'ignore', 'on_enter_overidden': 'ignore', 'on_exit_overidden': 'ignore', 'on_enter_constrained': 'ignore', 'on_exit_constrained': 'ignore', 'on_enter_blocked': 'ignore', 'on_exit_blocked': 'ignore'}
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] --------------------------------------------------
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Config other
2020-09-01 06:53:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Using DAY MODE parameters: {'delay': 600, 'service_data': None, 'service_data_off': None}
2020-09-01 06:53:12 INFO (MainThread) [custom_components.entity_controller] The entity_controller component is ready!
2020-09-01 06:53:32 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] 'binary_sensor.office_motion' Sensor state change to: off
2020-09-01 06:53:32 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state: 'idle'
2020-09-01 06:53:33 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state_entity_state_change :: [light.office_light] - old: None, new: <state light.office_light=off; friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:53:33.886638-04:00>
2020-09-01 06:53:33 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state_entity_state_change :: [light.office_light] - old: <state light.office_light=off; friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:53:33.886638-04:00>, new: <state light.office_light=on; brightness=100, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:53:33.887955-04:00>
2020-09-01 06:54:37 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] 'binary_sensor.office_motion' Sensor state change to: on
2020-09-01 06:54:37 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state: 'idle'
2020-09-01 06:54:37 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=100, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:53:33.887955-04:00>
2020-09-01 06:54:37 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 06:54:37 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=100, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:53:33.887955-04:00>
2020-09-01 06:54:37 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 06:54:37 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Exiting idle
2020-09-01 06:54:37 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_idle | Performing Transition Behaviour
2020-09-01 06:54:37 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_idle | Action - ignore
2020-09-01 06:54:37 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=100, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:53:33.887955-04:00>
2020-09-01 06:54:37 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 06:54:37 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_blocked | Performing Transition Behaviour
2020-09-01 06:54:37 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_blocked | Action - ignore
2020-09-01 06:55:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state_entity_state_change :: [light.office_light] - old: <state light.office_light=on; brightness=100, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:53:33.887955-04:00>, new: <state light.office_light=on; brightness=98, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:53:33.887955-04:00>
2020-09-01 06:55:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] significant attribute only change
2020-09-01 06:55:12 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=98, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:53:33.887955-04:00>
2020-09-01 06:55:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 06:55:13 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state_entity_state_change :: [light.office_light] - old: <state light.office_light=on; brightness=98, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:53:33.887955-04:00>, new: <state light.office_light=off; friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:13.293054-04:00>
2020-09-01 06:55:13 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=off; friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:13.293054-04:00>
2020-09-01 06:55:13 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are OFF.
2020-09-01 06:55:13 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_blocked | Performing Transition Behaviour
2020-09-01 06:55:13 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_blocked | Action - ignore
2020-09-01 06:55:13 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Entering idle
2020-09-01 06:55:13 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_idle | Performing Transition Behaviour
2020-09-01 06:55:13 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_idle | Action - off
2020-09-01 06:55:13 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_idle | Performing Action - Turning off
2020-09-01 06:55:13 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Turning off light.office_light
2020-09-01 06:55:13 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Calling service turn_off on light.office_light
2020-09-01 06:55:13 DEBUG (MainThread) [custom_components.entity_controller] Resetting state
2020-09-01 06:55:22 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state_entity_state_change :: [light.office_light] - old: <state light.office_light=off; friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:13.293054-04:00>, new: <state light.office_light=on; brightness=5, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>
2020-09-01 06:55:22 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state_entity_state_change :: [light.office_light] - old: <state light.office_light=on; brightness=5, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>, new: <state light.office_light=on; brightness=28, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>
2020-09-01 06:55:22 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] significant attribute only change
2020-09-01 06:55:23 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state_entity_state_change :: [light.office_light] - old: <state light.office_light=on; brightness=28, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>, new: <state light.office_light=on; brightness=103, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>
2020-09-01 06:55:23 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] significant attribute only change
2020-09-01 06:57:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] 'binary_sensor.office_motion' Sensor state change to: off
2020-09-01 06:57:12 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state: 'idle'
2020-09-01 06:57:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] 'binary_sensor.office_motion' Sensor state change to: on
2020-09-01 06:57:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state: 'idle'
2020-09-01 06:57:15 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=103, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>
2020-09-01 06:57:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 06:57:15 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=103, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>
2020-09-01 06:57:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 06:57:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Exiting idle
2020-09-01 06:57:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_idle | Performing Transition Behaviour
2020-09-01 06:57:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_idle | Action - ignore
2020-09-01 06:57:15 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=103, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>
2020-09-01 06:57:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 06:57:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_blocked | Performing Transition Behaviour
2020-09-01 06:57:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_blocked | Action - ignore
2020-09-01 06:58:56 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] 'binary_sensor.office_motion' Sensor state change to: off
2020-09-01 06:58:56 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state: 'blocked'
2020-09-01 06:59:01 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] 'binary_sensor.office_motion' Sensor state change to: on
2020-09-01 06:59:01 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state: 'blocked'
2020-09-01 06:59:01 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_blocked | Performing Transition Behaviour
2020-09-01 06:59:01 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_blocked | Action - ignore
2020-09-01 06:59:01 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=103, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>
2020-09-01 06:59:01 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 06:59:01 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_blocked | Performing Transition Behaviour
2020-09-01 06:59:01 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_blocked | Action - ignore
2020-09-01 07:00:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] 'binary_sensor.office_motion' Sensor state change to: off
2020-09-01 07:00:15 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state: 'blocked'
2020-09-01 07:00:48 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] 'binary_sensor.office_motion' Sensor state change to: on
2020-09-01 07:00:48 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state: 'blocked'
2020-09-01 07:00:48 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_blocked | Performing Transition Behaviour
2020-09-01 07:00:48 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_blocked | Action - ignore
2020-09-01 07:00:48 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=103, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>
2020-09-01 07:00:48 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 07:00:48 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_blocked | Performing Transition Behaviour
2020-09-01 07:00:48 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_blocked | Action - ignore
2020-09-01 07:02:34 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] 'binary_sensor.office_motion' Sensor state change to: off
2020-09-01 07:02:34 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state: 'blocked'
2020-09-01 07:02:41 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state_entity_state_change :: [light.office_light] - old: <state light.office_light=on; brightness=103, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>, new: <state light.office_light=on; brightness=98, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>
2020-09-01 07:02:41 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] significant attribute only change
2020-09-01 07:02:41 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=98, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>
2020-09-01 07:02:41 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 07:02:42 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state_entity_state_change :: [light.office_light] - old: <state light.office_light=on; brightness=98, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T06:55:22.638183-04:00>, new: <state light.office_light=off; friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:42.401215-04:00>
2020-09-01 07:02:42 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=off; friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:42.401215-04:00>
2020-09-01 07:02:42 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are OFF.
2020-09-01 07:02:42 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_blocked | Performing Transition Behaviour
2020-09-01 07:02:42 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_blocked | Action - ignore
2020-09-01 07:02:42 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Entering idle
2020-09-01 07:02:42 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_idle | Performing Transition Behaviour
2020-09-01 07:02:42 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_idle | Action - off
2020-09-01 07:02:42 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_idle | Performing Action - Turning off
2020-09-01 07:02:42 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Turning off light.office_light
2020-09-01 07:02:42 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Calling service turn_off on light.office_light
2020-09-01 07:02:42 DEBUG (MainThread) [custom_components.entity_controller] Resetting state
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] 'binary_sensor.office_motion' Sensor state change to: on
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state: 'idle'
2020-09-01 07:02:51 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=off; friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:42.401215-04:00>
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are OFF.
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Exiting idle
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_idle | Performing Transition Behaviour
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_idle | Action - ignore
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Using DAY MODE parameters: {'delay': 600, 'service_data': None, 'service_data_off': None}
2020-09-01 07:02:51 INFO (MainThread) [custom_components.entity_controller.ec_office_light] {'delay': 600, 'service_data': None, 'service_data_off': None}
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] light params before turning on: {'delay': 600, 'service_data': None, 'service_data_off': None}
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_active | Performing Transition Behaviour
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_active | Action - on
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_active | Performing Action - Turning on
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Turning on light.office_light (no parameters passed to service call)
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Calling service turn_on on light.office_light
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state_entity_state_change :: [light.office_light] - old: <state light.office_light=off; friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:42.401215-04:00>, new: <state light.office_light=on; brightness=5, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:51.782684-04:00>
2020-09-01 07:02:51 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=5, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:51.782684-04:00>
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 07:02:51 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=5, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:51.782684-04:00>
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] Turning off entities, cancelling timer
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_active | Performing Transition Behaviour
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_exit_active | Action - ignore
2020-09-01 07:02:51 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=5, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:51.782684-04:00>
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_blocked | Performing Transition Behaviour
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] on_enter_blocked | Action - ignore
2020-09-01 07:02:51 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=5, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:51.782684-04:00>
2020-09-01 07:02:51 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
2020-09-01 07:02:52 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] state_entity_state_change :: [light.office_light] - old: <state light.office_light=on; brightness=5, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:51.782684-04:00>, new: <state light.office_light=on; brightness=103, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:51.782684-04:00>
2020-09-01 07:02:52 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] significant attribute only change
2020-09-01 07:02:52 INFO (MainThread) [custom_components.entity_controller.ec_office_light] <state light.office_light=on; brightness=103, friendly_name=Office Light, icon=mdi:dome-light, supported_features=41 @ 2020-09-01T07:02:51.782684-04:00>
2020-09-01 07:02:52 DEBUG (MainThread) [custom_components.entity_controller.ec_office_light] State entities are ON. [light.office_light]
➜  entity_controller `

**Comment by danobot** [comment]
I managed to reproduce the issue and of course ran into the problem discussed in #98 once again. This time I came up with a crude solution. 
EC is blocking itself because it interprets its own calls to `turn_on` as external control. Since there is no way to determine _who_ triggered a state change with the HASS API (until attribution is implemented), I created a short 2-second grace period after turningg on entities where EC will ignore any state changes.

Please use proper formatting for logs or a paste bin next time. People really don't appreciate log dumps like that.

I am checking in a fix to develop branch, please load it into your HA instance and retest :)

**Comment by broyuken** [comment]
Sorry, I wasn't sure what the proper formating for logs was. I could edit and use pastebin if you'd like.

**Comment by danobot** [comment]
All good bud 👍 Let me know how the testing goes.

**Comment by broyuken** [comment]
Nope, it went active timer for a couple seconds then immediately went into blocked.

**Comment by broyuken** [comment]
I have to change the timeout to 5 seconds as I have some GE light switches that take ~3 seconds to turn on/off and update their state in HA. This caused them to immediately go into the blocked state like we saw before.

**Comment by broyuken** [comment]
So fairly large issue I found. The original issue of 3.2 seems fixed now after changing it to 5 seconds, however a new issue that I didn't notice before is happening. The lights never turn off, the timer goes on, it expires, the EC light goes back to idle but the light stays on. Then, the next time I walk into the room the EC light goes into blocked which will further keep it from ever turning off since the light was on already before motion was triggered. This happened before my change as well, which I confirmed by rolling back to your dev version. The lights were turning off properly when using the GA version.

**Comment by danobot** [comment]
I suggest testing EC in isolation using `input_booleans` specially created for this test. This is to avoid other EC instances getting triggered by the same motion sensor and accidentally interpreting this as a problem with the EC instance under test.

If your GE lights are slow to respond (more than 2 seconds) then I can see why EC would go into blocked state. It is only ignoring state changes for 2 seconds after the light is turned on. After two seconds it interprets any state change as manual control. I can make this 2 seconds configurable to better handle situation like these where there is a lot of latency in control entities.

**Comment by broyuken** [comment]
I have restarted HA without making any changes and things seem to be working as intended now, not sure what the cause of the earlier issues was.

**Comment by danobot** [comment]
Ok that sounds good enough to me. Raise new issues on GitHub for any other problems you find. I will push the release out soon.

Cheers

**Comment by broyuken** [comment]
Cheers and thank you for your help!
